src/authorizations/components/redesigned/TokenRow.tsx

```diff
@@ -72,7 +68,6 @@ class TokensRow extends PureComponent<Props> {
             onUpdate={this.handleUpdateName}
             onClick={this.handleClickDescription}
             name={description}
-            noNameString={DEFAULT_TOKEN_DESCRIPTION}
```

Comment:
IMO, we still need to keep this here. Our users may have tokens without any description from before this change and for that token we still want to display the default token description.